client/coral-sign-in/components/FakeComment.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
I believe it is, but why is an extra component necessary here? It seems like it cold be a pain to keep this up to date with the main comment component.